doc/user-guide.adoc

```diff
@@ -107,7 +107,7 @@ or configure the transaction subsystem to store transaction log in database with
 link:https://docs.wildfly.org/23/wildscribe/subsystem/transactions/index.html#attr-use-jdbc-store[use of JDBC store].
 ====
 
-A `volumeClaimTemplate` cna be specifed to configure `Resources` requirements to store WildFly standalone data directory.
+A `volumeClaimTemplate` cna be specified to configure `Resources` requirements to store WildFly standalone data directory.
```

Comment:
We forgot this one "**cna** be specified", for the sake of completeness, would you mind fixing it with your PR?

Reply:
Haha, nice one. Fixed.

Reply:
Thank you, @rhusar !